### DISCUSSION TITLE
Default playback Speed

### DISCUSSION THREAD

**Discussion by theksepyro** [PaulWoitaschek/Voice#1790] [Ideas]
The playback speed could be set as a default option for every book, that way one wouldn't have to manually adjust it for each book (or would only need to do fine adjustments)

**Comment by MNeMoNiCuZ** [comment]
Yes. This would be a great setting.

**Comment by ndas108** [comment]
The only feature I'm really missing. Useful for listening to lectures. Related to Global Settings feature request: #1989

**Comment by Inf-Sub** [comment]
This would be a great idea.
